### PR TITLE
Fission/commcare onboarding

### PR DESCRIPTION
**Pull request by hemanthReddy06**

### CODE REVIEW

File: dimagi/pages/templates/sections/commcare/product_onboarding/pricing.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        {% endblocktrans %}
+      </p>
+      <p class="align-center gap-bottom-base">
+        <a class="arrow" href="#contact-form-modal">
+          {% blocktrans %}
+            Contact our sales team
+          {% endblocktrans %}
+        </a>
+      </p>
+  </div> {# .content #}
```
Comment (biyeun) [inline dimagi/pages/templates/sections/commcare/product_onboarding/pricing.html:37]: indentation is a bit off and was confusing to read

File: dimagi/pages/templates/sections/commcare/product_onboarding/pricing/heading.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<div class="feature-row border-top accent-gray-bg">
+  <div class="grid-4 grid-valign-middle gap-bottom-line pad-top-line">
+    <div class="column">
+    </div>{# .column #}
+    <div class="column">
+      <h5 class="align-center uppercase green font-weight-medium">
```
Comment (biyeun) [inline dimagi/pages/templates/sections/commcare/product_onboarding/pricing/heading.html:6]: Echoing the previous comment I made about the double `<h5>` tags and a better semantic change.

File: dimagi/pages/templates/sections/commcare/product_onboarding/onboard/_base.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{% load i18n %}
+{% load dimagidotcom %}
+
+<div class="column">
+  <a class="tile feature-projects pad-vert-line pad-horz-line border-rounded-large {% block feature_class %}{% endblock %}">
+    <div class="align-center pad-vert-small {% block icon_class %}{% endblock %}">
+      {% block icon %}{% endblock %}
+    </div>
+
+    <h5 class="align-center uppercase font-weight-medium {% block title_class %}{% endblock %}">
```
Comment (biyeun) [inline dimagi/pages/templates/sections/commcare/product_onboarding/onboard/_base.html:10]: i probably should have noticed this when I copied everything over, but semantically this tag should be a `<h3>` (keep the current styling though) and the pricing element should be a `<div>` not a header tag... would you be able to update this?

Comment (hemanthReddy06) [reply dimagi/pages/templates/sections/commcare/product_onboarding/onboard/_base.html:10]: Updated, but for the pricing element <div> is looking kinda small.
onboarding

Comment (biyeun) [reply dimagi/pages/templates/sections/commcare/product_onboarding/onboard/_base.html]: @hemanthReddy06 you can apply the h5 class to the element to style it as an h5 but not semantically treat it as an h5 (two different things)

Comment (hemanthReddy06) [reply dimagi/pages/templates/sections/commcare/product_onboarding/onboard/_base.html]: @biyeun Updated can you take a look.

**Pull request merged**